keyboards/atreus62/keymaps/d4mation/keymap.c

```diff
@@ -0,0 +1,196 @@
+   * `-----------------------------------------'                `-----------------------------------------'
+   */
+
+  [_NUM] = LAYOUT(
+    _______, _______, KC_PSLS, KC_PAST, KC_PMNS, _______,                   _______, _______, KC_PSLS, KC_PAST, KC_PMNS, _______,
+    _______, KC_KP_7, KC_KP_8, KC_KP_9, KC_PPLS, _______,                   _______, KC_KP_7, KC_KP_8, KC_KP_9, KC_PPLS, _______,
+    _______, KC_KP_4, KC_KP_5, KC_KP_6, KC_PPLS, _______,                   _______, KC_KP_4, KC_KP_5, KC_KP_6, KC_PPLS, _______,
+    _______, KC_KP_1, KC_KP_2, KC_KP_3, KC_PENT, _______,                   _______, KC_KP_1, KC_KP_2, KC_KP_3, KC_PENT, _______,
+    _______, KC_KP_0, KC_KP_0, KC_PDOT, KC_PENT, _______, _______, _______, _______, KC_KP_0, KC_KP_0, KC_PDOT, KC_PENT, _______
+  ),
+
+  /* "Adjust" layer, only active if both "Lower" and "Raise" are active at the same time
+   * All unused keys are blanked out for this layer
+   * ,-----------------------------------------.                ,-----------------------------------------.
+   * |  NO  |  NO  |  NO  |  NO  |  NO  |  NO  |                |  NO  |  NO  |  NO  |  NO  |  NO  |  NO  |
+   * |------+------+------+------+------+------|                |------+------+------+------+------+------|
+   * |  NO  |RESET |  NO  |  NO  |  NO  |  NO  |                |  NO  |  NO  |  NO  |  NO  |  NO  |  NO  |
+   * |------+------+------+------+------+------|                |------+------+------+------+------+------|
+   * |  NO  |  NO  |  NO  |UC WIN|UC OSX|  NO  |,------.,------.|  NO  |Dvorak|Qwerty|  NO  |  NO  |  NO  |
+   * |------+------+------+------+------+------|| Swap ||      ||------+------+------+------+------+------|
+   * |  NO  |  NO  |  NO  |  NO  |  NO  |  NO  ||  to  || Swap ||  NO  |  NO  |  NO  |  NO  |  NO  |  NO  |
+   * |------+------+------+------+------+------|| Ctrl || Back ||------+------+------+------+------+------|
+   * |  NO  |  NO  |  NO  |  NO  |      |  NO  |`------'`------'|  NO  |      |  NO  |  NO  |  NO  |  NO  |
+   * `-----------------------------------------'                `-----------------------------------------'
+   */
+
+  [_ADJUST] = LAYOUT(
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, RESET,   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, XXXXXXX, XXXXXXX, UC_M_WI, UC_M_OS, XXXXXXX,                   XXXXXXX, DF(_DVR),DF(_QWR),XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, _______, XXXXXXX, CG_SWAP, CG_NORM, XXXXXXX, _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+  ),
+
+};
+
+/* Runs just one time when the keyboard initializes. */
+void matrix_init_keymap( void ) {
+  set_unicode_input_mode( UC_OSX );
+};
```

Comment:
It may be better to do this with `eeconfig_init_user`, so that it happens when eeprom is initialized or re-initialized. 

But that's user choice.

Reply:
Ah, just read the docs on that function and it turns out I misunderstood it. I thought it would _write to the EEPROM each time the keyboard initialized_ which I figured would be a bad idea. It seems that `eeconfig_init_user()` would just be a default/fallback of sorts if I'm now understanding it correctly.

Reply:
Correct.  It's called if the keyboard hasn't been flashed by QMK before, if you use the eeprom reset key, if you reboot with bootmagic lite, etc.    Basically, it lets you set the default eeprom values.